docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -23,28 +23,132 @@ Users are able to write their own custom access monitoring queries by querying t
 ### Configuration
 
 <Tabs>
+<TabItem scope={["cloud","team"]} label="Teleport Cloud">
+
+Teleport Access Monitoring is enabled by default for all Teleport Cloud users.
+
+</TabItem>
 <TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
 
 To enable Access Monitoring you need to update the Teleport Auth Service configuration:
 ```yaml
 auth_service:
```

Comment:
Can you put this whole configuration block in an includable file and include it in the Teleport configuration reference https://goteleport.com/docs/reference/config/